v2/dtos/responses/deviceservice.go

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package responses
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+)
+
+// AddDeviceServiceResponse defines the Response Content for POST DeviceService DTOs.
+// This object and its properties correspond to the AddDeviceServiceResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/AddDeviceServiceResponse
+type AddDeviceServiceResponse struct {
+	common.BaseWithIdResponse `json:",inline"`
+}
+
+// UpdateDeviceServiceResponse defines the Response Content for PUT DeviceService as pushed DTO.
+// This object and its properties correspond to the UpdateDeviceServiceResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/UpdateDeviceServiceResponse
+type UpdateDeviceServiceResponse struct {
+	common.BaseWithIdResponse `json:",inline"`
+}
```

Comment:
Remove `AddDeviceServiceResponse` & `UpdateDeviceServiceResponse` and just use common.BaseWithIdResponse. Update swagger to match.

Reply:
Removed and updated the swagger.